### ISSUE TITLE
Can see only nitrishape outline?

### ISSUE THREAD

**Issue by dkspwndj** [niftools/nifskope#129]
I still use blender 2.49b to converting mods.
So export to fallout 4 nif and changing nif user version.
Than deleting
BSshaderPPLightingProperty
NiSpecularProperty
NiMaterialProperty
and pasting skyrim texture property.
So removed texture data nitrishape cant see in object window and unable click.
In 1.1.3 than that object can see. Should it work like old version? or only outline to enable clicking.

**Comment by neomonkeus** [comment]
Marking as invalid.
Information provided is not adequate, and no reference nif provided.

**Comment by hexabits** [comment]
There's so much more wrong with this issue than missing files.  I can hardly understand it.

If it's invisible there is a reason.  Make sure you are using 12-27 release to rule out shader/renderer issues.  If you are already on 12-27,  press Alt+S to turn off shaders to rule out further shader issues.  Otherwise,  it's invisible because it has no material or some either criteria for being hidden,  which is how it's supposed to be.  To show hidden objects you need to turn on "Show Hidden". 

Furthermore you say export to FO4 which must be a mistake.  Further still, changing user version is not supported behavior.  The data alignment for NiAVObjects (shapes) is completely different between 34 (FO3) and 83 (Skyrim) so you create invalid data by swapping user versions because the same data gets reinterpreted with different data alignment.   That is if changing the user version still "worked", which it doesn't, as no one was ever supposed to do that anyway.

**Comment by neomonkeus** [comment]
Yeah, I only took a cursory glance and marked it as invalid given the information provided.
Didn't mean to imply that failing to provide a nif was the sole reason, I have updated my comment.
